.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "workbench.colorTheme": "Default Dark Modern",
```

Comment:
remove?

Reply:
This is the default for VSCode but not cursor and I wanted the same theme...